crates/typed-store/src/rocks/mod.rs

```diff
@@ -1825,14 +1864,40 @@ pub fn base_db_options() -> DBOptions {
     }
 
     // The table cache is locked for updates and this determines the number
-    // of shards, ie 2^6. Increase in case of lock contentions.
+    // of shards, ie 2^10. Increase in case of lock contentions.
     opt.set_table_cache_num_shard_bits(10);
 
+    // Allow buffering 1GiB of writes before flushing to disk.
+    opt.set_write_buffer_size(256 * 1024 * 1024);
+    opt.set_max_write_buffer_number(4);
```

Comment:
Would it make sense to make these env variables? I think longer term we likely need a config/yaml file to read these from and load during startup